specification.md

```diff
@@ -4,73 +4,73 @@ Opportunity:
+* Compose a _SOMA collection_ object out of pre-existing TileDB arrays.  This object is the composition of one or more _SOMAs_(terms defined below).
+* Simple access to SOMA/SOMA-collection properties (e.g., `obs`, `var`, `X`) and ability to slice/query on the entire object based on `obs`/`var` labels.
+* For Python, import/export an in-memory AnnData object (for subsequent use with AnnData/ScanPy) from _either_ a slice/query result or the entire object.  For R, same basic function but to/from Seurat and SingleCellExperiment.
 
 This draft, written by TileDB and CZI, proposes an API for single-cell data that attempts to unify the data models followed by [AnnData](https://anndata.readthedocs.io/en/latest/), Bioconductor’s [SingleCellExperiment](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html), [Seurat](https://github.com/satijalab/seurat/) and [CXG](https://github.com/chanzuckerberg/single-cell-explorer/blob/main/dev_docs/cxg.md). We are seeking community feedback.
 
-It first describes a general data model that captures all the above frameworks. Then it explains how TileDB can implement this model on-disk so that we have a concrete implementation reference. Next, it describes how TileDB can query this model with its generic array API “soon” (as some minor features are a work in progress). Subsequently, it proposes a more single-cell-specific API that can easily be built on top of TileDB’s, in order to hide the TileDB-specific implementation and API. We will focus only on Python for now for simplicity. Finally, it concludes with a list of features that will need to be implemented on the TileDB side in order to have a working prototype very soon.
+It first describes a general data model that captures all the above frameworks. Then it explains how TileDB can implement this model on-disk so that we have a concrete reference implementation. Next, it describes how TileDB can query this model with its generic array API “soon” (as some minor features are a work in progress). Subsequently, it proposes a more single-cell-specific API that can easily be built on top of TileDB's, in order to hide the TileDB-specific implementation and API. We will focus only on Python for now for simplicity. Finally, it concludes with a list of features that will need to be implemented on the TileDB side in order to have a working prototype very soon.
 
 ## The Data Model
 
-Based on our research and avoiding a lot of the jargon, there are the following “components” in typical single-cell projects, which account for various means of representing filtered & transformed datasets, and associating data from different assays/modalities.
+Based on our research and avoiding a lot of the jargon, there are the following “components” in typical single-cell projects, which account for various means of representing filtered & transformed datasets, and associating data from different assays/modalities:
```

Comment:
I'm doing a light review but if I was rewriting, I'd be eliminating as much text as possible to make it easier for reviewers and readers:

~Based on our research and avoiding a lot of the jargon~ - we don't have to show our math. Simply make the points and allow reviewers to disagree if we missed part of the model.